routes/event.js

```diff
@@ -315,6 +358,70 @@ function sendUpdateEmail(event) {
 	});
 }
 
+function addGuestDetails(event, details, callback) {
+	var guestDetails = details;
+
+	for (var i = 0; i < event.invitees.length; i++)
+	{
+		if (event.invitees[i].state == 'Attending')
+		{
+			guestDetails = guestDetails + 'Guest Email: ' + event.invitees[i].email + '\nGuest Access Requirements: ' + event.invitees[i].accessRequirements + '\nGuest Dietary Restrictions: ' + event.invitees[i].dietaryRestrictions + '\n\n';
```

Comment:
Shouldnt add guest if their requirements are null and would be worthwhile to add their name in too along with their email. Since the table plan I'd assume would be by name not email

Reply:
![image](https://user-images.githubusercontent.com/16799316/37941075-0e21fde4-3164-11e8-83ce-ae42d789ed77.png)